From: Yves-Alexis Perez <corsac@debian.org>
Date: Mon, 14 Jan 2019 13:38:45 +0100
Subject: Inconditionnaly include err.h and errno.h

HAVE_SYSCTL is false on new architectures like arm64 or x32, and build
will fail (#766766).
---
 panel-plugin/libacpi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/panel-plugin/libacpi.c b/panel-plugin/libacpi.c
index 0755585..1789cba 100644
--- a/panel-plugin/libacpi.c
+++ b/panel-plugin/libacpi.c
@@ -54,10 +54,10 @@ static int      acpifd;
 
 #ifndef __linux__
 #include <sys/sysctl.h>
-#include <err.h>
-#include <errno.h>
 #endif
 
+#include <err.h>
+#include <errno.h>
 #endif
 
 #include "libacpi.h"
